openshift_performance/ci/scripts/bulk_delete.sh

```diff
@@ -0,0 +1,67 @@
+#/!/bin/bash
+#set -x
+################################################
+## Auth=prubenda@redhat.com
+## Desription: Script for running bulk delete
+## of empty and loaded projects and comparing times
+################################################
+file_1="../content/conc_proj.yaml"
+file_2="../content/pyconfigLoadedProject.yaml"
+delete_output=bulk_delete.out
+function create_projects()
+{
+  python ../../../openshift_scalability/cluster-loader.py -f $1
+}
+
+function delete_projects() {
+  ./delete_projects.sh
+}
+
+function rewrite_yaml() {
+  python3 -c "import increase_pods; increase_pods.print_new_yaml($1,'$2')"
+}
+
+python3 -m pip install ruamel.yaml
+
+rm -if $delete_output
+
+exit 0
```

Comment:
This halts the script.   Debug artifact?

Reply:
yes definitely not supposed to be there....deleting